client/src/components/ProjectWizard/SidebarPoints/SidebarPointsPanel.js

```diff
@@ -15,6 +15,16 @@ const SidebarPointsPanel = props => {
     earnedPointsRule = rules.filter(rule => rule.code === "PTS_EARNED")[0];
     projectLevelRule = rules.filter(rule => rule.code === "PROJECT_LEVEL")[0];
   }
+
+  const rulesConfig = {
```

Comment:
Yep, this is a great way to get both the target and earned values in one object!